components/templates/src/lib.rs

```diff
@@ -47,8 +47,15 @@ pub fn load_tera(path: &Path, config: &Config) -> Result<Tera> {
 
     // Only parsing as we might be extending templates from themes and that would error
     // as we haven't loaded them yet
-    let mut tera =
-        Tera::parse(&tpl_glob).context("Error parsing templates from the /templates directory")?;
+    let mut tera = match Tera::parse(&tpl_glob) {
```

Comment:
We have the path from above. We can just join `templates` and see if it exists. If it does then we load Tera with the glob otherwise Tera::default()

Reply:
That would be a little more straight forward.